js-packages/@prestojs/ui-antd/src/widgets/SelectAsyncChoiceWidget.tsx

```diff
@@ -375,10 +393,14 @@ function SelectAsyncChoiceWidget<
                 // from happening.
                 return;
             }
+            setLastValue([value]);
             return onChange(value.value);
         },
         [onChange]
     );
+
+    const missingLabel =
+        value && (Array.isArray(value) ? value.find(v => v.missingLabel) : value.missingLabel);
```

Comment:
`value.find(v => v.missingLabel)` would give you the `v`, not the label itself. do you mean 
`value.find(v => v.missingLabel)?.missingLabel`?

Reply:
ah, looking at how its used, is this only intended as a check? if thats the case might be better to use `.some` instead of `.find` (less confusing)

Reply:
we don't actually need the value, just need to know whether it exists. i've changed it to `isMissingLabel` and converted to boolean value